constraints.txt

```diff
@@ -1,5 +1,6 @@
 cftime<1.1.1;python_version=='3.5'
+cftime<1.5;python_version=='3.8'
```

Comment:
I was actually testing this. probably better to merge the 1st and 2nd line into :
```
cftime<1.5;python_version<='3.8'
```

Reply:
That might break things for 3.5 - I assume there was a reason to pin to `<1.1.1` previously!?

Reply:
oh my bad, didn't see the diff in version number!

Reply:
But maybe it should be 
```
cftime<1.1.1;python_version=='3.5'
cftime<1.5
```